ext/date/php_date.c

```diff
@@ -2363,6 +2363,11 @@ static HashTable *date_object_get_properties_timezone(zval *object) /* {{{ */
 	return props;
 } /* }}} */
 
+static inline void date_object_update_properties_timezone(zval *object) /* {{{ */
```

Comment:
just for clarification - you decided to create new inline function just to have more obvious function name, am I right?